checks.d/dns_check.py

```diff
@@ -52,21 +52,27 @@ def query_dns(self, instance, timeout):
             resolver.nameservers = [nameserver]
 
         record_type = instance.get('record_type', 'A')
+        expected_results = instance.get('addresses', [])
 
         status = AgentCheck.CRITICAL
         start_time = time.time()
         try:
             self.log.debug('Querying "%s" record for hostname "%s"...' % (record_type, hostname))
             answer = resolver.query(hostname, rdtype=record_type)
+            resolved_value = answer.rrset.items[0].to_text()
```

Comment:
What about multiple DNS records ? We should probably takes all the addresses and try to see if one of them matches (all of them maybe makes more sense). What do you think ?